src/common/base_client.ts

```diff
@@ -401,4 +411,78 @@ export abstract class BaseClient {
             await this.userSubscribe(this.userSubscribeParams);
         }
     }
+
+    private getRequestError(code: number, data?: any) : KolibriRequestError {
+        let error: KolibriRequestError;
+        switch (code) {
+            case -31901:
+                error = new GeneralError(data);
```

Comment:
to save the error variable and the break line you could directly call "return new GeneralError(data)". But its also ok like it is. :)